lib/pipeline.js

```diff
@@ -17,6 +17,7 @@ class PipelineModel extends BaseModel {
      * Create a pipeline
      * @method create
      * @param  {Object}   config                Config object to create the pipeline with
+     * @param  {Object}   config.owners         The owners of this repository
```

Comment:
@stjohnjohnson Good catch. I was debating if it should be named `owners` or `admins` and decided `admins` makes more sense but forgot to change the comment.